packages/components/src/custom-select-control-v2/styles.ts

```diff
@@ -13,6 +13,13 @@ import { chevronIconSize } from '../select-control/styles/select-control-styles'
 import { fontSizeStyles } from '../input-control/styles/input-control-styles';
 import type { CustomSelectButtonSize } from './types';
 
+// TODO: extract to common utils and apply to relevant components
+const ANIMATION_PARAMS = {
+	SLIDE_AMOUNT: '2px',
```

Comment:
Should this be increased in order to make it more obvious that the select options are dropping down?

Reply:
Now that you changed it to starting from `- SLIDE_AMOUNT`, it looks good actually 👍 Nice touch!